fe/fe-core/src/main/java/org/apache/doris/common/util/TimeUtils.java

```diff
@@ -190,16 +197,18 @@ public static synchronized Date parseDate(String dateStr, PrimitiveType type) th
         if (!matcher.matches()) {
             throw new AnalysisException("Invalid date string: " + dateStr);
         }
+        dateStr = formatDateStr(dateStr);
```

Comment:
Do we still need `synchronized` for this method?

Reply:
Methods using DateTimeFormatter do not need to be synchronized